docs/_snippets/_direct_observability_integration_options.md

```diff
@@ -0,0 +1,38 @@
+import Image from '@theme/IdealImage';
+import AdvancedDashboard from '@site/static/images/cloud/manage/monitoring/advanced_dashboard.png';
+import NativeAdvancedDashboard from '@site/static/images/cloud/manage/monitoring/native_advanced_dashboard.png';
+
+### Direct Grafana plugin integration {#direct-grafana}
+
+The ClickHouse data source plugin for Grafana enables visualization and exploration of data directly from ClickHouse using system tables. This approach works well for monitoring performance and creating custom dashboards for detailed system analysis.
+For plugin installation and configuration details, see the ClickHouse data source plugin. For a complete monitoring setup using the Prometheus-Grafana mix-in with pre-built dashboards and alerting rules, see Monitor ClickHouse with the new Prometheus-Grafana mix-in.
```

Comment:
"see the ClickHouse data source plugin" -> add a link
"see Monitor ClickHouse with the new Prometheus-Grafana mix-in." -> add a link